src/simplewallet/simplewallet.cpp

```diff
@@ -9690,6 +9864,8 @@ bool simple_wallet::wallet_info(const std::vector<std::string> &args)
     type = tr("Watch only");
   else if (m_wallet->multisig(&ready, &threshold, &total))
     type = (boost::format(tr("%u/%u multisig%s")) % threshold % total % (ready ? "" : " (not yet finalized)")).str();
+  else if (m_wallet->is_background_wallet())
+    type = tr("Background wallet");
```

Comment:
Good spot. In the latest I use the term "background wallet" for background wallets, and "background syncing wallet" for a normal wallet that is background syncing. How's that?

Reply:
That's interesting, I didn't even become aware during review that there are *two* terms for *two* different things, I was assuming mere "sloppiness" regarding terminology, so to say ...

Two terms that are not as similar as those two could be something, but I wasn't able to come up with something in this regard. I think it will be fine with those two used consequently.